src/test/java/com/thindeck/dynamo/DyTasksTest.java

```diff
@@ -66,8 +72,109 @@ public void getTask() throws Exception {
             Matchers.equalTo(tid)
         );
     }
+    
+    /**
+     * Test that validate the add method with attributes in NULL
+     * @throws Exception In case of error
+     */
+    @Test
+    public void addTaskNullAttributes() throws Exception{
+    	final Repo repo = new MkRepo();
+    	final Region region = DyTasksTest.region(repo.name());
+    	final String command = "command";
+    	
+    	DyTasks tasks = new DyTasks(region, repo);
+    	Task task = tasks.add(command,null);
+    	
+    	MatcherAssert.assertThat(task.command(),Matchers.equalTo(command));   	
+    }
 
     /**
+     * Test that validate the add method with empty attributes
```

Comment:
Description should be something like "DyTask can add a task without attributes."